### PR TITLE
Add custom theme/syntax detection to info.sh

### PR DESCRIPTION
**Pull request by eth-p**

### CODE REVIEW

File: diagnostics/info.sh [hunk 1]
```diff
@@ -39,6 +41,19 @@ _tool_:description() {
 _bat_:run() {
 	_out bat --version
 	_out env | grep '^BAT_\|^PAGER='
+
+	local cache_dir="$(bat --cache-dir)"
+	if [[ -f "${cache_dir}/syntaxes.bin" ]]; then
+		_print_command "bat" "--list-languages"
+		echo "Found custom syntax set."
+		bat --list-languages
```
Comment (sharkdp) [inline diagnostics/info.sh]: Hm, do we really want to include these large lists in the output?

I think the presence of a custom syntax/theme set is an important factor, but the actual content will probably rarely be.

Comment (eth-p) [reply diagnostics/info.sh]: That's a fair point. Maybe a diff instead, or just echo the presence of one?

Comment (sharkdp) [reply diagnostics/info.sh]: I think the presence alone would be enough for now.

**Comment by eth-p** [comment]
I amended the commit to no longer list the syntax/theme sets, but only the presence of them.

**Comment by sharkdp** [comment]
Thank you!

**Pull request merged**